data/ipl3.cfg

```diff
@@ -197,6 +197,10 @@ astraAllStarAstroNN = $MWM_ASTRA/{v_astra}/summary/astraAllStarAstroNN-{v_astra}
 astraAllVisitAstroNN = $MWM_ASTRA/{v_astra}/summary/astraAllVisitAstroNN-{v_astra}.fits
 astraAllStarApogeeNet = $MWM_ASTRA/{v_astra}/summary/astraAllStarApogeeNet-{v_astra}.fits
 astraAllVisitApogeeNet = $MWM_ASTRA/{v_astra}/summary/astraAllVisitApogeeNet-{v_astra}.fits
+astraAllStarApogeeNetv2 = $MWM_ASTRA/{v_astra}/summary/astraAllStarApogeeNetv2-{v_astra}.fits
+astraAllVisitApogeeNetv2 = $MWM_ASTRA/{v_astra}/summary/astraAllVisitApogeeNetv2-{v_astra}.fits
+astraAllStarBossNet = $MWM_ASTRA/{v_astra}/summary/astraAllStarBossNet-{v_astra}.fits
+astraAllVisitBossNet = $MWM_ASTRA/{v_astra}/summary/astraAllVisitBossNet-{v_astra}.fits
```

Comment:
Given that you have a lot of related file species that only vary by a name, I would suggest we consolidate all these paths down by adding more template keywords.  Something like

```
astraAllStar = $MWM_ASTRA/{v_astra}/summary/astraAllStar{pipeline}-{v_astra}.fits
astraAllVisit = $MWM_ASTRA/{v_astra}/summary/astraAllVisit{pipeline}-{v_astra}.fits
```
to replace all the separate allStar and allVisit files.  These are really all one type of data product anyway.    If think this can also be done with the `astra` pipeline output paths, e.g.
```
astraStar = $MWM_ASTRA/{v_astra}/results/star/@sdss_id_groups|/astraStar{pipeline}-{v_astra}-{sdss_id}@component_default|.fits
astraVisit = $MWM_ASTRA/{v_astra}/results/visit/@sdss_id_groups|/astraVisit{pipeline}-{v_astra}-{sdss_id}@component_default|.fits

This means you can also flexibly add pipelines or change pipeline names and we don't need to always update the tree.

```

Reply:
No Brian, these are only superficially the "same" species but each has a different data model!  They are different species with different datamodels, so collectivizing their paths would not be helpful for the data products database.